Fit-it-day placeholder update for Deploy to Production (Van) (#4895)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/reactnative/deployment.md

## Deploy a React Native app

> Mobile apps built for production are usually a binary file that you upload to an app store.

Reviewer: ```suggestion
A mobile app built for production is usually a binary file that you upload to an app store.
```
